src/Utils/GalleryUtils.ts

```diff
@@ -118,22 +137,38 @@ export function reportAbuse(
           throw new Error(`Received HTTP ${response.status} when submitting report for ${data.name}`);
         }
 
-        NotificationConsoleUtils.logConsoleInfo(
-          `Your report on ${data.name} has been submitted. Thank you for reporting the violation.`
+        dialogHost.showOkModalDialog(
+          "Report Abuse",
+          `Your report on ${data.name} has been submitted. Thank you for reporting the violation.`,
+          "OK",
+          undefined,
+          {
+            percentComplete: 1,
+          }
         );
+
         onComplete(response.data);
       } catch (error) {
         handleError(
           error,
           "GalleryUtils/reportAbuse",
           `Failed to submit report on ${data.name} violating code of conduct`
         );
-      }
 
-      clearSubmitReportNotification();
+        dialogHost.showOkModalDialog(
+          "Report Abuse",
+          `Failed to submit report on ${data.name} violating code of conduct`,
```

Comment:
>violating code of condu [](start = 51, length = 23)

same here

Reply:
I'll note it down and have a PR with all string updates later on. Trying to check this in asap to get feedback.